addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -157,15 +152,15 @@ const MoveArticleToDialog = Dialog.extend({
              * @param {Function} escapeMarkup
              */
             formatResult: (result, container, query, escapeMarkup) => {
-                const { text } = result;
+                const { text, subject } = result;
                 const markup = [];
                 window.Select2.util.markMatch(text, query.term, markup, escapeMarkup);
                 if (result.id === 'private') {
                     const src = escapeMarkup(this.getLoggedUserPicture());
                     markup.unshift(`<img src="${src}" class="rounded-circle mr-1"/>`);
-                } else if (typeof result.icon !== 'undefined') {
-                    const icon = result.icon || '📄';
-                    markup.unshift(`<span class="mr-1">${escapeMarkup(icon)}</span>`);
+                }
+                if (subject && subject != text) {
```

Comment:
Linting: `!==`

(Your linter should yell about that, make sure it's correctly configured).